vortex_controller/src/controller_ros.cpp

```diff
@@ -69,15 +59,30 @@ void Controller::commandCallback(const vortex_msgs::PropulsionCommand& msg)
   setpoints->update(time, command);
 }
 
-void Controller::stateCallback(const nav_msgs::Odometry &msg)
+ControlMode Controller::getControlMode(const vortex_msgs::PropulsionCommand& msg) const
+{
+  ControlMode new_control_mode;
+  new_control_mode = control_mode;
```

Comment:
it is not obvious when reading this function where `control_mode` originates from. Could we rewrite with
 `this->control_mode` maybe?

Reply:
I agree.